content/en/service_management/service_level_objectives/time_slice.md

```diff
@@ -0,0 +1,97 @@
+1. Select **By Time Slices** to define your SLo measurement. 
+1. Define your uptime condition by choosing a metric query, comparator and threshold. For example, to define uptime as whenever p95 latency is less than 1s. Alternatively, you can [import the uptime from a monitor](#import-from-a-monitor).
+1. Choose your timeframe and target
+1. Name and tag your SLO.
+1. Click **Create**.
+
+### Export an existing monitor SLO
+
+<div class="alert alert-info">Only metric monitor SLOs can be exported. Non-metric monitors or multi-monitor SLOs cannot be exported</div>
+
+Create a Time Slice SLO by exporting an existing Monitor-based SLO. From a monitor SLO click **Export to Time Slice SLO**.
+
+{{< img src="service_management/service_level_objectives/time_slice/export_monitor_slo.png" alt="On a Monitor-based SLO detail side panel, the button to Export to Time Slice is highlighted" style="width:90%;" >}}
+
+### Import from a monitor
+
+<div class="alert alert-info">Only metric monitor SLOs appear in the monitor selection for import. </div>
+
+From the Create or Edit SLO page, under *Define your SLI*, click **Import from Monitor** and select from the dropdown or search in the monitor selector.
+
+**Note**: Time Slice SLOs do not support rolling periods. Rolling periods do not transfer from a monitor query to a Time Slice query. 
+
+{{< img src="service_management/service_level_objectives/time_slice/import_from_monitor.png" alt="Highlighted option to Import From Monitor in the Define your SLI section of an SLO configuration" style="width:90%;" >}}
+
+## Uptime calculations
+
+To calculate the uptime percentage for a Time Slice SLOs, Datadog cuts the timeseries into equal-duration intervals, called "slices". The length of the interval is 5 minutes and not configurable. The space and time aggregation are determined by the metric query. For more information on space and time aggregation, see the [metrics][1] documentation for. 
+
+For each slice, there is a single value for the timeseries, and the uptime condition (such as `value < 1`) is evaluated for each slice. If the condition is met, the slice is considered uptime, otherwise it is considered downtime.
+
+### Groups and overall uptime
+
+Time Slice SLOs allows you to track uptime for individual groups, where groups are defined in the "group by" portion of the metric query. 
+
+When groups are present, uptime is calculated for each individual group. However, overall uptime works differently. In order to match existing monitor SLO functionality, Time Slice SLOs use the same definition of overall uptime. When **all** groups have uptime, it is considered overall uptime. Conversely, if **any** group has downtime, it is considered overall downtime. Overall uptime is always less than the uptime for any individual group.
+
+<!-- In the example above, environment "staging" has 5 minutes of downtime over a 24-hour period, resulting in approximately 99.652% of uptime.
```

Comment:
@roxanne-moslehi Realized this was commented out, I couldn't find these examples from the SLO manual, but if you have other examples, send me the PNG screenshots or a link to the SLOs and I can add an example use case!

Reply:
@estherk15 will check this and follow up with you